### PR TITLE
Use std::vector for colwidths

### PR DESCRIPTION
**Pull request by tobiolo**
Use std::vector as columns are modified (inserted/removed) dynamically.

### CODE REVIEW

**Comment by tobiolo** [comment]
This aims to make use of conveniences provided by the STL and thus to increase maintainability of the code

File: src/grid.h [hunk 1]
```diff
@@ -5,7 +5,7 @@ struct Grid {
     // subcells
     Cell **cells;
     // widths for each column
-    int *colwidths {nullptr};
+    std::vector<int> colwidths {};
```
Comment (aardappel) [inline src/grid.h:8]: `{}` not needed here.

**Review by aardappel — COMMENTED** [review]
nice simplification :)

**Comment by tobiolo** [comment]
You are welcome :+1:

File: src/grid.h (hunk 1 above)
Comment (tobiolo) [reply src/grid.h:8]: Ah thanks, pushed a commit for this.

**Pull request merged**